gossip/src/crds_value.rs

```diff
@@ -485,6 +498,69 @@ impl Sanitize for NodeInstance {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Default, PartialEq, Eq, AbiExample)]
+pub struct RestartLastVotedForkSlots {
+    pub slots: EpochSlots,
+    pub last_voted_slot: Slot,
+    pub last_voted_hash: Hash,
+}
+
+impl Sanitize for RestartLastVotedForkSlots {
+    fn sanitize(&self) -> std::result::Result<(), SanitizeError> {
+        self.slots.sanitize()?;
+        self.last_voted_hash.sanitize()
+    }
+}
+impl fmt::Debug for RestartLastVotedForkSlots {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "RestartLastVotedForkSlots {{ slots: {:?} last_voted: {}({}) }}",
+            self.slots, self.last_voted_slot, self.last_voted_hash
+        )
+    }
+}
```

Comment:
Why do we need this?
Can't it be just `#[derive(Debug)]`?

Reply:
Removed.